### ISSUE TITLE
fyi: compilation, abi, deploy soon to be supported by jsonrpc

### ISSUE THREAD

**Issue by heikoheiko** [SilentCicero/meteor-dapp-cosmo#1]
https://github.com/ethereum/go-ethereum/issues/774

**Comment by SilentCicero** [comment]
Will integrate on 774. Although there are good aspects to using SOLJS, i.e. compiling will still work if unconnected with geth node.

**Comment by step21** [comment]
yeah ... especially as geth for some reason likes to not see my solc compiler, even when specifying at startup.
